addons/auth_password_policy/models/res_users.py

```diff
@@ -27,7 +27,7 @@ def _check_password_policy(self, passwords):
             if not password:
                 continue
             if len(password) < minlength:
-                failures.append(_("Passwords must have at least %d characters, got %d.", minlength, len(password)))
+                failures.append(_("Your password must contain at least %d characters and only has %d.", minlength, len(password)))
```

Comment:
Could be opportunity to use named parameters, easing translation process (even if this one should be understandable, always better to be sure), like
```suggestion
                failures.append(_("Your password must contain at least %(minimal_length)d characters and only has %(current_count)d.", minlength, len(password)))
```